src/types.js

```diff
@@ -1400,9 +1400,21 @@ class ResourceNode {
    *  the resource.
    */
   fullPropertyName() {
+    // check if the property name is for a primitive extension in FHIR
+    let propName = (this.propName?.startsWith('_') && this.model) ?
```

Comment:
I think after the changes in the `children()` function, these lines can be replaced with:
`let propName = this.propName;`